bc_obps/service/addresses_service.py

```diff
@@ -29,8 +19,12 @@ class AddressesData:
 
     @classmethod
     def upsert_addresses_from_data(
-        cls, address_data: AddressesData, physical_address_id, mailing_address_id, prefix=""
-    ):
+        cls,
+        address_data: DictStrAny,
```

Comment:
Both part II reg and removing the `class AddressesData` because it was not checking the type. Additionally, we are using prefixes like `{prefix}physical_street_address`, so we are not certain about the exact data we will receive from `address_data`.